The-Chonky-Panda/Atelier
# Add new review modal

## Description

Skeleton for `Write Your Review` modal.

## How to test

Manual test:
on click of `ADD YOUR REVIEW` button, a modal should show up on the bottom with a few input fields.
on click of `Add Review`/`Cancel` button in the modal, the modal should close up.

## Notes


## Issue tracking

https://trello.com/c/EX7Nsc2n
https://trello.com/c/Ul2dgE3B
https://trello.com/c/Jvn3HkJ3
